mainflux/things.py

```diff
@@ -191,3 +247,41 @@ def disconnect(self, thing_id: str, channel_id: str, token: str):
                 errors.things["disconnect"], http_resp.status_code
             )
         return mf_resp
+    
+    def share_thing(self, user_id: str, channel_id: str, actions: dict, token: str):
+        """Share thing"""
+        payload = {"object": channel_id, "subject": user_id, "Actions": actions, "External": True}
+        http_resp = requests.post(
+            self.URL + "/policies",
+            headers=utils.construct_header(token, utils.CTJSON),
+            data=json.dumps(payload)
```

Comment:
You don't need to do a `json.dumps`
```suggestion
            data=payload
```